packages/ketcher-core/src/application/editor/Editor.ts

```diff
@@ -428,6 +459,20 @@ export class CoreEditor {
     this.sequenceTypeEnterMode = mode;
   }
 
+  private onChangeToggleIsSequenceSyncEditMode(
+    isSequenceSyncEditMode: boolean,
+  ) {
+    if (!(this.mode instanceof SequenceMode)) {
+      return;
+    }
+
+    if (isSequenceSyncEditMode) {
+      this.mode.turnOnSyncEditMode();
+    } else {
+      this.mode.turnOffSyncEditMode();
+    }
+  }
```

Comment:
I assume `toggle` function shouldn't accept any value but just negate the current state of the control (compared to `set` function)
I think it would fit your logic as well, rather then explicitly setting the exact value when clicking the sync button you can just switch to the opposite value within sequence mode